### PR TITLE
Feature TR-1440: forward port synchronization improvements

### PR DESCRIPTION
**Pull request by wazelin**
# [TR-1440](https://oat-sa.atlassian.net/browse/TR-1440)

This brings a hotfix into the current version of the extension.

### [v3.2.0.1](https://github.com/oat-sa/extension-tao-encryption/releases/tag/v3.2.0.1) synchronization improvement
No cherry picking done here, [`backport/depp-offline`](https://github.com/oat-sa/extension-tao-encryption/compare/backport/depp-offline) has been used as a foundation.

### CODE REVIEW

File: model/Task/DecryptDeliveryExecutionTask.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?php
```
Comment (siwane) [inline model/Task/DecryptDeliveryExecutionTask.php:1]: missing declare strict type and legal notice

File: model/Task/DecryptDeliveryExecutionTask.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function jsonSerialize()
+    {
+        return __CLASS__;
+    }
+}
```
Comment (siwane) [inline model/Task/DecryptDeliveryExecutionTask.php:41]: missing new line at end of file

File: model/Task/DecryptDeliveryExecutionTask.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace oat\taoEncryption\Task;
+
+use oat\oatbox\extension\AbstractAction;
+use oat\taoEncryption\Service\Result\DecryptResultService;
+
+class DecryptDeliveryExecutionTask extends AbstractAction
```
Comment (siwane) [inline model/Task/DecryptDeliveryExecutionTask.php:8]: Why to copy/paste this class? It creates lot of duplicate and only the array keys change

Comment (wazelin) [reply model/Task/DecryptDeliveryExecutionTask.php:8]: Those are all commits from the past ;)

I'm just trying to align the [_latest_ known-to-work extension version](https://github.com/oat-sa/extension-tao-encryption/releases/tag/v3.2.0.1) with the current release branch. That "backport" was _never_ merged into `develop`.

See [this comparison](https://github.com/oat-sa/extension-tao-encryption/compare/v3.2.0...v3.2.0.1) for reference (spoiler, it's exactly the contents of this PR).

Comment (wazelin) [reply model/Task/DecryptDeliveryExecutionTask.php:8]: Perhaps, that port is not even needed then. I'm just aligning the version that was actually used with the current release.

**Review by siwane — APPROVED** [review]

**Pull request merged**